InfluxDb/InfluxDb.php

```diff
@@ -0,0 +1,5 @@
+<?php namespace App\SupportedApps\InfluxDb;
```

Comment:
The correct name is `InlfuxDB` please rename `InfluxDb` to `InfluxDB`.
```suggestion
<?php namespace App\SupportedApps\InfluxDB;
```